### DISCUSSION TITLE
31. October 2023

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#1625] [Meeting notes]
### Agenda

*(please write your proposed agenda items in comments under this discussion)*

- [x] @woju: SCAG (Scaffolding project) update

### Woju: scag-client demo (end-to-end Gramine enclave development and attestation solution)

Recently pushed v0.1 of the Scaffolding project.

[ shows the demo ]

### Woju: proposal to adopt repo to https://github.com/gramineproject GitHub organisation

Woju wants Scaffolding to live in the Gramineproject org.

https://gramine.readthedocs.io/en/latest/index.html#protect-your-application -- Scaffolding is under Protect Application, but Mona & Benny argue that this should be under Protect Container (and maybe rename the section names, as Scaffolding is about the *final* Docker container, not the original container/app).

When Gramine v1.6 is released, Scaffolding will also be marked as v1.6. After that, Scaffolding tracks the Gramine version (just like GSC currently).

Scaffolding will have the same review rules as other Gramine repos.

**Comment by woju** [comment]
- [x] woju: SCAG (Scaffolding project) update:
  - `scag-client` demo (end-to-end Gramine enclave development and attestation solution)
  - proposal to adopt repo to https://github.com/gramineproject GitHub organisation
